streams/src/main/java/org/apache/kafka/streams/state/internals/CachingKeyValueStore.java

```diff
@@ -61,7 +59,7 @@
     public void init(final ProcessorContext context,
                      final StateStore root) {
         initInternal(context);
-        underlying.init(context, root);
+        super.init(context, root);
```

Comment:
You'll note in all these methods that we need to say `super.***` to explicitly invoke the operation on the wrapped store when it's a method that the subclass explicitly overrides.